app/collections/playlists.js

```diff
@@ -1,9 +1,47 @@
 import Backbone from 'backbone';
 import Playlist from '../models/playlist';
+import cozysdk from 'cozysdk-client';
 
 
 const Playlists = Backbone.Collection.extend({
-    model: Playlist
+   	
+   	model: Playlist,
+
+    initialize: function() {
+        this.on('add', this.onAdd, this);
+    },
+
+    onAdd: function(playlist) {
+        if (!playlist.get('_id')) {
```

Comment:
I don't understand why you force saving on the playlist when you add it to the collection. If you want to be sure the playlist is registered on the server, why not using the `create` method on the collection with the unsaved model object?